articles/article-117-Why-stick-to-one-example.md

```diff
@@ -0,0 +1,29 @@
+<!-- BEGIN TITLE -->
+# Why stick to one example
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Using the same example, again and again, seems to be quite boring without any added benefit.
+In some instances, this can be used for some positive effect. Using a known example to show different things (like methods or algorithms) saves time, as the people don't have to think about the example used in much detail.
```

Comment:
Language suggestion: In some instance, however, this ....  (to make the message stronger)